source/options.tsx

```diff
@@ -9,7 +9,7 @@ import {assertError} from 'ts-extras';
 import * as indentTextarea from 'indent-textarea';
 import delegate, {DelegateEvent} from 'delegate-it';
 import {isChrome, isFirefox} from 'webext-detect-page';
-import {isEnterprise} from 'github-url-detection';
```

Comment:
`isEnterprise` is always true in the options, because "the location is not GitHub.com" 🤦‍♂️ 

I fixed this detection